models/events/Event.proto

```diff
@@ -40,9 +39,8 @@ message Event {
         models.events.PredictedTrainData predicted_train = 14;
         models.events.RailwayUndertakingData railway_undertaking = 15;
         models.events.StationData station = 16;
-        models.events.TrainDelayNotification train_delay_notification = 17;
-        models.events.TrainInformationData train_information = 18;
-        models.events.VehicleData vehicle = 19;
-        models.events.WeatherData weather = 20;
+        models.events.TrainInformationData train_information = 17;
+        models.events.VehicleData vehicle = 20;
```

Comment:
müsste nach der 17 nicht 18 kommen oder sieht das in der git ansicht nur so aus?